src/Integration/Resources/Strings.resx

```diff
@@ -296,7 +296,7 @@ ocalization note: '[try again]()' is a special markup used to translate the text
     <comment>Button text for syncing an already bound SonarQube project to a VS solution</comment>
   </data>
   <data name="ShowAllProjectsCommandText" xml:space="preserve">
-    <value>Show all projects</value>
+    <value>Show All Projects</value>
```

Comment:
Make each* word upper-case to be consistent with the rest of VS.
*Except words like "in" and "as".